include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1527,12 +1521,17 @@ int bt_le_adv_stop(void);
 /**
  * @brief Create advertising set.
  *
- * Create a new advertising set and set advertising parameters.
- * Advertising parameters can be updated with @ref bt_le_ext_adv_update_param.
+ * Create an instance for independent advertising with its own parameters and data.
+ * The advertising set remains valid until deleted with @ref bt_le_ext_adv_delete.
+ * Advertising parameters can be updated with @ref bt_le_ext_adv_update_param, and advertising
+ * can be started with @ref bt_le_ext_adv_start.
```

Comment:
A few other places in this PR the `()` approach is used instead of the `@ref` for function. We could consider being consistent